vcd/config_test.go

```diff
@@ -1111,4 +1116,14 @@ func skipNoNsxtConfiguration(t *testing.T) {
 	if testConfig.VCD.NsxtProviderVdc.StorageProfile == "" {
```

Comment:
Not related to tests, but I'm thinking about previous addition:
> * Add support for NSX-T Org VDC [GH-550]

Should we have a comment/link/explanation about it in the external network resource docs? Right now, it's very easy to miss the fact that VDC supports NSX-T. It's better if the docs represent the new feature explicitly.

Reply:
> Should we have a comment/link/explanation about it in the external network resource docs? Right now, it's very easy to miss the fact that VDC supports NSX-T. It's better if the docs represent the new feature explicitly.

Did you mean inside external network docs or org vdc docs? Should I post a `note` in org vdc?

Reply:
Maybe even in both a sentence?

Reply:
I have added a note to `vcd_org_vdc` but wondering if we should put reference into external network. One does not need to have org vdc created to create external network so these resources are not directly related.

Reply:
Perfect, thanks!